wof/wofpy_config.py

```diff
@@ -25,11 +28,13 @@
 
 Options:
   -h --help     Show this screen.
-  -v --version     Show version.
+  -v --version  Show version.
+  --mode=test   deploy mode [default: test]
 """
 
-_ROOT = os.path.abspath(os.path.join(os.pardir, os.path.dirname(__file__)))
-_CONFIG = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
+_ROOT = os.path.abspath(os.path.dirname(wof.__file__))
+_CONFIG = os.path.join(_ROOT, 'examples', 'production_configs')
+_ODM2_TIMESERIES = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
```

Comment:
Note that in the source code we have `odm2/timeseries` instead of `odm2timeseries`. Maybe we should consider renaming there too for to improve code consistency.